### PR TITLE
Allow Reuse of Chips

### PR DESCRIPTION
**Pull request by jamesmcclain**

### CODE REVIEW

File: catalogs/README.md [hunk 1]
```diff
@@ -31,10 +31,10 @@ If your project requires additional python dependencies, add them to the `requir
...
 | Sen1Floods11                      | s3://sen1floods11-data/stac-catalog.zip                                                            | Sen1Floods11 dataset (https://github.com/cloudtostreet/Sen1Floods11)                 |
-| Sen1Floods11 s1weak ML            | s3://sen1floods11-data/mldata_s1weak_sen1.tar.gz                                                   | Sen1Floods11 imagery + labels (weakly trained s1 labels)                             |
-| Sen1Floods11 s2weak ML            | s3://sen1floods11-data/mldata_s2weak_sen1.tar.gz                                                   | Sen1Floods11 imagery + labels (weakly trained s2 labels)                             |
-| Sen1Floods11 training w/ HAND     | s3://sen1floods11-data/mldata_hand_sen1.tar.gz                                                     | Sen1Floods11 imagery with HAND + Sen1Floods11 labels                                 |
+| Sen1Floods11 s1weak ML            | s3://sen1floods11-data/mldata_s1weak_sen1.tar.gz                                                   | Sen1Floods11 imagery (w/ weak labels derived from Sentinel-1)                        |
```
Comment (jamesmcclain) [inline catalogs/README.md:34]: @moradology I changed the descriptions of this and the next two STACs in this table, and the location of the last STAC in this table.

File: pipeline/usfimr_raster_pipeline.py [hunk 2]
```diff
@@ -169,7 +170,7 @@ def build_dataset_from_catalog(catalog: Catalog, class_config: ClassConfig,
     return DatasetConfig(
         class_config=class_config,
         train_scenes=train_scenes,
-        validation_scenes=validation_scenes,
+        validation_scenes=(validation_scenes + test_scenes),
```
Comment (moradology) [inline pipeline/usfimr_raster_pipeline.py:173]: I'm curious about the exclusion of a `test_scenes` here

**Review by moradology — APPROVED** [review]
Everything looks reasonable, though some of the upstream changes for/in RV that this depends on aren't 100% obvious to me

File: pipeline/usfimr_raster_pipeline.py (hunk 2 above)
Comment (jamesmcclain) [reply pipeline/usfimr_raster_pipeline.py:173]: I am including `test_scenes` here.  I just want them to be part of raster vision's validation set.

**Pull request merged**